xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -284,15 +275,9 @@ void CGUIDialogPVRTimerSettings::InitializeSettings()
   if (tag->SupportsFolders())
     AddEdit(group, SETTING_TMR_DIR, 19076, 0, tag->m_strDirectory, true, false, 19104);
 
-  AddToggle(group, SETTING_TMR_RADIO, 19077, 0, tag->m_bIsRadio);
```

Comment:
could you please check where the label 19077 is used and in case this was the only reference also remove it? Unfortunately the label doesn't have a comment on where it's used so a quick search in code will be needed.